components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -316,21 +317,14 @@ public boolean isThisType(RandomAccessInputStream stream) throws IOException {
       return buf;
     }
 
-    in.seek(offset + planeSize * no);
-
-    int tile = series;
-    for (int i=0; i<index; i++) {
-      tile -= tileCount[i];
-    }
-
-    // seek instead of skipBytes to prevent dangerous int cast
-    in.seek(in.getFilePointer() + tile * planeSize * getImageCount());
-    in.skipBytes(bytesToSkip * getSizeY() * no);
+    seekStartOfPlane(no, offset, planeSize);
 
     if (bytesToSkip == 0) {
       readPlane(in, x, y, w, h, buf);
     }
     else {
+      // seek instead of skipBytes to prevent dangerous int cast
```

Comment:
Slightly confused here: "instead of skipBytes" but appears to use skipBytes? :+1: for comments though!